node/cn/filters/api.go

```diff
@@ -79,6 +82,7 @@ func NewPublicFilterAPI(backend Backend, lightMode bool) *PublicFilterAPI {
 		chainDB: backend.ChainDB(),
 		events:  NewEventSystem(backend.EventMux(), backend, lightMode),
 		filters: make(map[rpc.ID]*filter),
+		timeout: deadline,
```

Comment:
Can we rename deadline -> defaultTimeout or filterTimeout? It's not deadline anymore.

Reply:
thank you. added at here: https://github.com/klaytn/klaytn/pull/1851/commits/b59d8c7fe0718627d65f1922a0d6a94948225de9